From 4c71b9478ffc83a0d6129ea3347bd628b129b780 Mon Sep 17 00:00:00 2001
From: Patrick Delaunay <patrick.delaunay@foss.st.com>
Date: Tue, 19 Dec 2023 10:19:28 +0100
Subject: [PATCH 0757/1141] dt-bindings: serial: fix power-domains property

Serial is a power-domain-consumer defined in power-domain-consumer.yaml [1]

it is NOT a generic PM domain provider define in power-domain.yaml

Fix dtbs_check warning:
power-domains: [[8]] is not of type 'object'

[1] is defined in :
https://github.com/devicetree-org/dt-schema/blob/main
=> dtschema/schemas/power-domain/power-domain.yaml

Signed-off-by: Patrick Delaunay <patrick.delaunay@foss.st.com>
Change-Id: Ic23029ab189a5d7aeee1cb60ff60193b31c66f9f
Reviewed-on: https://gerrit.st.com/c/mpu/oe/st/linux-stm32/+/347851
Reviewed-by: Amelie DELAUNAY <amelie.delaunay@foss.st.com>
Domain-Review: Amelie DELAUNAY <amelie.delaunay@foss.st.com>
Reviewed-by: Valentin CARON <valentin.caron@foss.st.com>
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
---
 Documentation/devicetree/bindings/serial/st,stm32-uart.yaml | 1 -
 1 file changed, 1 deletion(-)

--- a/Documentation/devicetree/bindings/serial/st,stm32-uart.yaml
+++ b/Documentation/devicetree/bindings/serial/st,stm32-uart.yaml
@@ -67,7 +67,6 @@ properties:
   rs485-rx-during-tx: true
 
   power-domains:
-    $ref: ../power/power-domain.yaml
     maxItems: 1
 
   rx-threshold:
